honeybee_radiance/cli/translate.py

```diff
@@ -58,8 +67,14 @@ def model_to_rad_folder(model_json, folder, view, grid, config_file, minimal, lo
         if folder is None:
             folder = os.path.dirname(os.path.abspath(model_json))
 
-        # re-serialize the Model and translate the model to a radiance folder
+        # re-serialize the Model and perform any checks
         model = Model.from_file(model_json)
+        if not no_grid_check and len(model.properties.radiance.sensor_grids) == 0:
```

Comment:
Does the model always have `properties.radiance` attribute? I imagine you can have cases where the model is generated without honeybee-radiance being installed and this attribute will be missing.

You may want to add a check for `hasattr(model.properties, 'radiance')` before checking the length of sensor grids. Same for the view part.

Reply:
As long as `honeybee-radiance` is installed, all Python model objects will have `properties.radiance`.  If there were no radiance properties in the Model JSON that was loaded, then all of the radiance properties are the default.

So there's no need for an extra `hasattr` check here.